app/web/test/utils.tsx

```diff
@@ -1,8 +1,5 @@
 import { fireEvent, screen } from "@testing-library/react";
 import mediaQuery from "css-mediaquery";
-import { TFunction } from "i18n";
-import { StringMap, TOptions } from "i18next";
-import i18n from "test/i18n";
```

Comment:
They stopped exposing `StringMap` purposely (says in their docs). I tried to re-write this function but it was getting pretty gnarly, which is why I decided to just use the test instance of i18n in the tests and use the t function from there rather than mocking a separate one.